server/index.js

```diff
@@ -74,34 +74,41 @@ app.get('/api/cart/', (req, res, next) => {
   where "c"."cartId" = $1
   `;
   const params = [cartId];
+  if (!req.session.cartId) { res.status(200).json([]); }
```

Comment:
If you don’t include a return statement, the route will continue trying to query the database